definitions/nodes.md

```diff
@@ -1,13 +1,20 @@
 ---
 title: Nodes
 status: Feedback Appreciated
-category: concept
+category: Concept
 ---
+
 ## Nodes
 
-A node is a computer that works in concert with other computers, or nodes, to accomplish a common task. When we talk about cloud native platforms a node could be a physical computer, a virtual computer, referred to as a VM, or even a container.
+### What it is
+
+A node is a computer that works in concert with other computers, or nodes, to accomplish a common task. When we talk about cloud native platforms a node could be a physical computer, a virtual computer, referred to as a [VM](virtual_machine.md), or even a [container](container.md).
+
+### Problem it Addresses
+
+A node exists to run apps or processes as part of a group of computers. A node is part of a [cluster](cluster.md), or group, of nodes that works together in concert.
```

Comment:
This is not a problem definition, it's rather still part of the _what it is_. 

Here's my attempt. @JasonMorgan, let's review together. Not sure if this is the right answer here. 

While an application could (and many do) run on one single machine, there are some risks involved with that. Processes that share a machine aren't properly separated and, if one process crashes, it can drag other processes down with it. To address this, developers started creating [distributed applications](distributed_apps.md) where each process runs on its own node.

Reply:
Maybe link to cluster as well as they're deeply related.